storage/redis/redisnamespace/redisnamespace.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package redisnamespace
+
+import "strconv"
+
+var dbs = map[string]int{
+	"live-accounting.db":      0,
+	"server.revocation-dburl": 1,
+}
```

Comment:
why a map here instead of `const`?

Reply:
I wanted to be able to iterate over them and you can't (as far as I know) do that with `const`.